guides/source/asset_pipeline.md

```diff
@@ -1177,28 +1177,8 @@ TIP: For further details have a look at the docs of your production web server:
 Assets Cache Store
 ------------------
 
-The default Rails cache store will be used by Sprockets to cache assets in
```

Comment:
You can still configure the cache store, se only need to change this code to use the proper API. It is like the last example given in the previous text.